src/config/index.ts

```diff
@@ -282,6 +282,7 @@ export class Config extends (EventEmitter as {
 					}`,
```

Comment:
For the `throw new Error` a few lines above it might make sense to use a `TypeError`